### ISSUE TITLE
Update MIXS ReadMe, Documentation new Packages, Expected Values 

### ISSUE THREAD

**Issue by lschriml** [GenomicsStandardsConsortium/mixs#82]
Review MIXS ReadMe, update content 

Environmental package: MIXS-Ag
Structured comment name: 20 character limit, no spaces, utilize underscores in the naem: e.g. collection_date
Package item: full term name, e.g. collection date
Definition: A description of the package item, including links to ontologies or other sources that can be used to fill in the package item values. 
Expected value: Short description and/or expected value for a package item.

Value syntax: The proper syntax for writing the value of a given package item
Options, as listed above,  include: {text}, {PMID}|{DOI}|{URL},  {integer}, {termLabel} {[termID]},  {float} {unit}, 		
For enumerations: list pipe delimited options in alphabetical order, such as: [eukaryote|bacteria_archaea]

Requirement: mandatory (M), conditional mandatory (C), optional (X), environment-dependent (E) or not applicable(-) for a given package or checklist item
Preferred units: a unit suggestion if a measurement value is given
e.g.  meter, millimeter, gram, liter, degree Celcius, year, percent, parts per million

Occurrence: indicates whether a given package or checklist item can be used only once (1), multiple times (m), or none (0)
MIXS ID: a unique ID for the item. Terms re-used from other packages or checklists will be assigned the same MIXS ID. 

Item Formatting: 
Dates: YYYY-MM-DD, e.g. 2020-02-26
Geographic locations: country name:region, specific location name, e.g.  USA:Maryland, Baltimore

If multiple values of a package item are allowed to be reported, this is indicated in the definition and under occurrence (m). 
The multiple terms are pipe delimited.   E.g  greenhouse|laboratory 

Updated Documentation for new Package organization, 
 
  Package organization: [order of items]
      MIXS Core
     -reused MIXS terms
    - New Package terms
           - mandatory
            - conditionally mandatory & optional

Updated Documentation for Expected Values 

  Expected values                                       Value syntax                        Examples
A controlled vocabulary list                            
 e.g. eukaryote, bacteria_archaea
enumeration			                     	
                                                                    [eukaryote|bacteria_archaea]

decimal degrees 				           {float} {float}                               	50.586825 6.408977
measurement value				   {float} {unit}			                        100 meter
country name:region, specific 
     location name     	                          {term}:{term}, {term}            	        USA:Maryland, Baltimore
date and time					          {timestamp}			                        2018-05-11
[Ontology defined terms]

Include an explanation of the 
ontology to use, And the terms/IDs.
   E.g. env_broad_scale 

Add terms that identify the major environment     	{termLabel} {[termID]}	            	forest biome [ENVO:01000174]
type(s) where your sample was collected. 
Recommend subclasses of biome 
[ENVO:00000428].

PATO					                                      {termLabel} {[termID]}	             	allopolyploidy [PATO:0001379]		
enumeration 				                                     [air|built environment]                     	soil
[used when provided a vocabulary of terms
The list is alphabetized]

number of base pairs				                     {integer} bp			                        300000 bp	

PMID, DOI or URL				                            {PMID}|{DOI}|{URL}	                               doi:10.1016/j.syapm.2018.01.009

names of organisms that the entity		           {text}				                              human, animal, plant
 is pathogenic to

gene name					                            {text}				                             16S rRNA, 18S rRNA

**Comment by ramonawalls** [comment]
Should be documented in [PR](https://github.com/GenomicsStandardsConsortium/mixs/tree/606-mixs-editing-workflow-and-policy) 606

**Comment by only1chunts** [comment]
I think this is a duplicate of #496 so I'm closing this ticket
